plugins/bulk-import/src/components/PreviewFile/PreviewPullRequestForm.tsx

```diff
@@ -308,10 +267,11 @@ export const PreviewPullRequestForm = ({
         variant="outlined"
         margin="normal"
         fullWidth
-        name={`repositories.${pullRequest[repoId]?.componentName}.prTitle`}
-        value={pullRequest[repoId]?.prTitle}
+        name={`${repoId}.prTitle`}
+        value={formik.values?.[repoId]?.prTitle}
```

Comment:
Wouldn't it be easier if you can remove the `repoId` and the current formik state contains just information of the current form?

Reply:
I initially started with this thought, but the complexity arises in the tabbed view